version-history.md

```diff
@@ -9,6 +9,7 @@ permalink: /version-history/
 
 OpenSearch version | Release highlights | Release date
 :--- | :--- | :--- | :---
+[1.3.0]() | Adds Model Type Validation to Validate Detector API, continuous transforms, custom actions, applied policy parameter to explain API, default action retries, and new rollover and transition conditions to Index Management, new New ML Commons plugin, parse command to SQL, Application Analytics, Live Tail, Correlation, and Events Flyout to Observbility, and auto backport and support for OPENSEARCH_JAVA_HOME to Performance Analyzer. Bug fixes. | 17 March 2022
```

Comment:
"new New ML Commons plugin" is this intentional

Reply:
Also, a nit, but what's our policy on capitalizing API names? You have "Validate Detector API" (caps), and then later "explain API" (lower).

Reply:
Apparently I was on something while writing this